lektor/db.py

```diff
@@ -18,12 +21,12 @@
      iteritems, range_type
 from lektor import metaformat
 from lektor.utils import sort_normalize_string, cleanup_path, \
-     untrusted_to_os_path, fs_enc
+     untrusted_to_os_path, fs_enc, locate_executable
 from lektor.sourceobj import SourceObject, VirtualSourceObject
 from lektor.context import get_ctx, Context
 from lektor.datamodel import load_datamodels, load_flowblocks
 from lektor.imagetools import (
-    ThumbnailMode, make_thumbnail,
+    ThumbnailMode, make_image_thumbnail,
```

Comment:
Nitpicking: I would leave this as is, and `from imagetools import make_thumbnail as  make_image_thumbnail` (and do the same with `videotools.make_thumbnail`). In the idea that at some point in the future this code might converge.

Reply:
I think changing this makes it easier to grep for where things are used. It matches other functions such as `get_image_info` and `process_image` as well.

Though I have to admit there are  some inconsistencies (`get_suffix`, `get_quality`).